src/components/Menu/Menu.component.jsx

```diff
@@ -0,0 +1,34 @@
+import { Drawer } from '@material-ui/core';
+import React from 'react'
+import List from '@material-ui/core/List';
```

Comment:
El import de React lo puedes quitar ya no es necesario en las nuevas versiones. Tienes más así en otros archivos así dejo solo este comment al respecto para no andar de castor con las demás lineas.

```suggestion
import { Drawer } from '@material-ui/core';
import List from '@material-ui/core/List';
```

Reply:
x2, salvo que ocupes un Hook, seria bueno no tener ese import de React

Reply:
No se puede joven, da error. Así se va a quedar